torch/csrc/autograd/python_variable.cpp

```diff
@@ -1,6 +1,4 @@
 #include <ATen/NamedTensorUtils.h>
-#include <ATen/core/PythonFallbackKernel.h>
```

Comment:
Any more headers we can remove by any chance? :)

Reply:
Yeah looks like there are some. I'll follow up in another PR

Reply:
Sure sounds good